### PR TITLE
Jack Rename

### PR DESCRIPTION
**Pull request by moodmosaic**
This pull request renames all occurrences of `Jack` to `Hedgehog`, except those found in

- URLs on the README
- the Seed module's comments

(Let me know if I could also do a pull request for Haskell and one for PureScript.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 [![Build Status](https://travis-ci.org/jystic/dotnet-jack.svg?branch=master)](https://travis-ci.org/jystic/dotnet-jack)
 
-# dotnet-jack
+# dotnet-Hedgehog
```
Comment (jystic) [inline README.md]: casing

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
 An alternative property-based testing system for F#, in the spirit of John Hughes & Koen Classen's [QuickCheck](https://web.archive.org/web/20160319204559/http://www.cs.tufts.edu/~nr/cs257/archive/john-hughes/quick.pdf).
 
 The key improvement is that shrinking comes for free — instead of generating a random value and using a shrinking function after the fact, we generate the random value and all the possible shrinks in a rose tree, all at once.
 
 ![](https://github.com/moodmosaic/dotnet-jack/raw/master/img/dice.jpg)
 
-*Jack's love of dice has brought him here, where he has taken on the form of an F# library, in order to help you gamble with your properties.*
+*Hedgehog's love of dice has brought him here, where he has taken on the form of an F# library, in order to help you gamble with your properties.*
```
Comment (jystic) [inline README.md]: maybe just remove this whole line, doesn't make sense anymore

File: README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
 ## Table of Contents
 
 * [Highlights](#highlights)
 * [Getting Started](#getting-started)
   * [At a glance](#at-a-glance)
-  * [Integrated shrinking](#-integrated-shrinking-is-an-important-quality-of-jack)
+  * [Integrated shrinking](#-integrated-shrinking-is-an-important-quality-of-Hedgehog)
```
Comment (jystic) [inline README.md]: casing

File: README.md (hunk 2 above)
Comment (moodmosaic) [inline README.md]: Yeah, I think so.

**Pull request merged**